modules/formulize/templates/admin/screen_settings.html

```diff
@@ -1,153 +1,152 @@
+<{* securitytoken should be part of the form *}>
+<{* formulize_admin_handler and formulize_admin_key are required, to tell what the name of the save handling file is, and what the key is that we're inserting/updating on *}>
+
+<form id="form-admin-form" class="formulize-admin-form">
+<{$securitytoken}>
+<input type="hidden" name="formulize_admin_handler" value="screen_settings">
+<input type="hidden" name="formulize_admin_key" value="<{$content.sid}>">
+<input type="hidden" name="formulize_admin_fid" value="<{$content.fid}>">
+<input type="hidden" name="aid" value="<{$content.aid}>">
+<input type="hidden" id="delete_passcode" name="delete_passcode" value="">
+<input type="hidden" id="make_new_passcode" name="make_new_passcode" value="">
+<input type="hidden" id="add_existing_passcode" name="add_existing_passcode" value="">
+
+
+
+<div class="panel-content content">
+  <fieldset>
+    <legend>Settings for the Screen: <em><{$content.title}></em></legend>
+
+	  <div class="form-item">
+		  <label for="screens-title">Name of the screen:</label>
+		  <input type="text" id="screens-title" name="screens-title" value="<{$content.title}>"/>
+	  </div>
+	  <div class="form-item">
+		  <label for="screens-type">What kind of screen is this:</label>
+
+	    <select id="screens-type" size="1" name="screens-type"<{if $content.sid neq 'new'}> disabled<{/if}>>
+		    <option value="listOfEntries"<{if $content.type eq 'listOfEntries'}> selected="selected"<{/if}>>List</option>
+		    <option value="multiPage"<{if $content.type eq 'multiPage'}> selected="selected"<{/if}>>Form</option>
+          <option value="template"<{if $content.type eq 'template'}> selected="selected"<{/if}>>Template</option>
+          <option value="calendar"<{if $content.type eq 'calendar'}> selected="selected"<{/if}>>Calendar</option>
+	    </select>
+			<{if $content.sid neq 'new'}>
+				<input type="hidden" name="screens-type" value="<{$content.type}>">
+			<{/if}>
+	  </div>
+  </fieldset>
+
+	<fieldset>
+		<legend>URLs for this screen</legend>
+		<p>The default URL for this screen: <a href="<{$xoops_url}>/modules/formulize/index.php?sid=<{$content.sid}>" target="_blank"><{$xoops_url}>/modules/formulize/index.php?sid=<{$content.sid}></a></p>
+		<p>Set an alternate URL for accessing this screen: <{$xoops_url}>/<input type='text' name='screens-rewriteruleAddress' value='<{$content.rewriteruleAddress}>'</p>
+		<div class="description">For alternate URLs to work, you will need to add code similar to this, to the .htaccess file at the root of your website:
+			<blockquote class="code"><span style="font-style: normal">
+			RewriteEngine On<br>
+			RewriteCond %{REQUEST_FILENAME} !-f<br>
+			RewriteCond %{REQUEST_FILENAME} !-d<br>
+			RewriteCond %{REQUEST_FILENAME} !-l<br>
+			RewriteRule ^(.*)$ /modules/formulize/index.php?formulizeRewriteRuleAddress=$1 [L]<br>
+			</span></blockquote>
```

Comment:
This messaging will always appear on every screen config regardless of if you've correctly set up the feature or not. Seems unnecessary.

I much prefer the drupal approach of "enabling" clean url support provided the server passes a check. Couldn't the feature be enabled on the formulize preferences screen where this configuration help text could also live? or is there a limit to what's possible since it's a xoops based configuration screen?

Reply:
Done in 68986a678958f9c8a36723f328e0aa5c48dec98d and 9e5426ee04cb96a8b9a6fbed4baf384a4068f353